homeassistant/components/azuredns/__init__.py

```diff
@@ -0,0 +1,176 @@
+"""Support for Azure DNS services."""
+import logging
+
+from datetime import datetime
+from datetime import timedelta
+
+import requests
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_HOST, CONF_DOMAIN
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+REQUIREMENTS = ['adal==1.2.1', 'aiohttp==3.5.4']
```

Comment:
No need requires `aiohttp` since it is part of core requirements

Reply:
Ok thanks. Also ran "gen_requirements_all.py" again. (See commit 9c3f8ba2e2d3a4972779a797470718a9fcf1231a)